### PR TITLE
Fix scripts to support python 3

### PR DESCRIPTION
**Pull request by Pro**
Also adds check on Linux for CI, to explicitly test python2 and python3

### CODE REVIEW

File: tests/fuzz/fuzz_binary_decode.cc [hunk 1]
```diff
@@ -21,7 +21,7 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 	size_t ptrSize = size;
 
 	// get some random type
-	uint8_t typeIndex = ptr[0];
+	uint16_t typeIndex = ptr[0];
```
Comment (HolgerJeromin) [inline tests/fuzz/fuzz_binary_decode.cc]: Is this python 3 related?

Comment (Pro) [reply tests/fuzz/fuzz_binary_decode.cc]: No, but I found it because there were never more than 255 types before.
so not related to python3, that's why it is in a separate commit
